### PR TITLE
Fixes "styles scoped" and "time indicator"

### LINKED ISSUE #74: scss is not scoped
The scss is not scoped, causing problems with styling when embedding Kalendar in a website.

### PR DESCRIPTION
**Pull request by mserralta**
Fixes #74 

Scoped styles

### CODE REVIEW

**Comment by mserralta** [comment]
Fixes #126

File: src/lib-components/kalendar-event.vue [hunk 1]
```diff
@@ -131,8 +131,10 @@ $creator-content: white;
         border: solid 1px white !important;
     }
 
-    &.inspecting {
```
Comment (altinselimi) [inline src/lib-components/kalendar-event.vue:134]: can you explain what width: 75! and transform scale(115%) achieves here.

File: src/lib-components/kalendar-container.vue [hunk 2]
```diff
@@ -314,7 +314,16 @@ export default {
         getTime,
         changeDay(numDays) {
             this.current_day = addDays(this.current_day, numDays).toISOString();
-            setTimeout(() => this.$kalendar.buildWeek());
```
Comment (altinselimi) [inline src/lib-components/kalendar-container.vue:317]: You should remove the 'dayChanged' commit from this PR, there is already a PR for this change.

File: src/lib-components/kalendar-weekview.vue [hunk 3]
```diff
@@ -177,9 +178,10 @@ export default {
       ]);
     },
     addHelperMethods() {
-      this.$kalendar.buildWeek = () => {
```
Comment (altinselimi) [inline src/lib-components/kalendar-weekview.vue:180]: Same as https://github.com/altinselimi/kalendar/pull/121/files#r1060817338

File: src/lib-components/kalendar-weekview.vue [hunk 4]
```diff
@@ -117,15 +118,15 @@ export default {
...
-      let { day_starts_at, day_ends_at, now } = this.kalendar_options;
+      let { day_starts_at, day_ends_at, now, cell_height } = this.kalendar_options;
       let time = getLocaleTime(now);
       let day_starts = `${time.split("T")[0]}T${(day_starts_at + "").padStart(2, '0')}:00:00.000Z`;
       let day_ends = `${time.split("T")[0]}T${(day_ends_at + "").padStart(2, '0')}:00:00.000Z`;
       let time_obj = new Date(time);
 
       if(new Date(day_ends) < time_obj || time_obj < new Date(day_starts)) return null;
 
-      let distance = (time_obj - new Date(day_starts)) / 1000 / 60;
```
Comment (altinselimi) [inline src/lib-components/kalendar-weekview.vue:128]: Im guessing this addresses some bug with passedTime which happens when you modify cell height ?

File: src/lib-components/kalendar-event.vue (hunk 1 above)
Comment (altinselimi) [inline src/lib-components/kalendar-event.vue:134]: can you explain what width: 75! and transform scale(115%) achieves here.

File: src/lib-components/kalendar-container.vue (hunk 2 above)
Comment (altinselimi) [inline src/lib-components/kalendar-container.vue:317]: You should remove the 'dayChanged' commit from this PR, there is already a PR for this change.

File: .gitignore [hunk 5]
```diff
@@ -1,6 +1,5 @@
 .DS_Store
 node_modules
-/dist
```
Comment (altinselimi) [inline .gitignore:3]: Please remove the /dist from the PR also.

**Review by altinselimi — CHANGES_REQUESTED** [review]
Great improvements. Though some feedback:
1) **Time indicator bug** and **Style scoped** should be separate PR
2) Need some explanation with what some code achieves

**Pull request closed without merging**